src/olympia/search/filters.py

```diff
@@ -471,12 +472,38 @@ def filter_queryset(self, request, qs, view):
         order_by = None
 
         if sort_param is not None:
+            split_sort_params = sort_param.split(',')
             try:
                 order_by = [self.SORTING_PARAMS[name] for name in
-                            sort_param.split(',')]
+                            split_sort_params]
             except KeyError:
                 raise serializers.ValidationError('Invalid "sort" parameter.')
 
+            # Random sort is a bit special.
+            # First, it can't be combined with other sorts.
+            if 'random' in split_sort_params and len(split_sort_params) > 1:
+                raise serializers.ValidationError(
+                    'The "random" "sort" parameter can not be combined.')
+
+            # Second, for perf reasons it's only available when the 'featured'
+            # param is present (to limit the number of documents we'll have to
+            # apply the random score to) and a search query is absent
+            # (to prevent clashing with the score functions coming from a
+            # search query).
+            if sort_param == 'random':
+                is_random_sort_available = (
```

Comment:
This should be `is_featured_param_available` or am I looking at this the other way around?

Reply:
No, I stand by that naming :) The variable is there to check if random sort is available, following the conditions described in the comment above.

Reply:
hmm, then I was looking at it the other way around, I'm fine with keeping it.